malloc.c

```diff
@@ -352,6 +352,12 @@ GC_API GC_ATTR_MALLOC void * GC_CALL GC_malloc(size_t lb)
     return GC_malloc_kind(lb, NORMAL);
 }
 
+/* Allocate lb bytes of composite (pointerful) data.    */
```

Comment:
I think I'm too stupid to understand this comment :) Can we document it more clearly, including spelling out what "unmanaged" means here?

Reply:
Admittedly this was a copy/paste job of the `GC_malloc` function above. Try this: 406442d03f64605abc7895c4341222d10c2e933a